web-components/src/components/footer/index.tsx

```diff
@@ -141,20 +49,31 @@ const useStyles = makeStyles<Theme, StyleProps>((theme: Theme) => ({
   },
 }));
 
+const sxs = {
+  text: {
+    color: 'primary.main',
+    mb: {
+      xs: 4.5,
+      sm: 3.75,
+    },
+  } as SxProps,
```

Comment:
In a prior PR I mentioned invoking these `as const`. While that works to pass the TS compiler, it doesn't provide type completion. Coercing into `SxProps` does, so I think that's preferable